Library/Formula/kibana.rb

```diff
@@ -0,0 +1,63 @@
+class Kibana < Formula
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://download.elastic.co/kibana/kibana/kibana-4.0.2-darwin-x64.tar.gz"
```

Comment:
That's what we're asking about: is it possible to make this use Homebrew's node instead of the packaged one?